pages/contact-us.html

```diff
@@ -0,0 +1,104 @@
+        <link rel="stylesheet" href="../blocks.css"/>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@tailwindcss/ui@latest/dist/tailwind-ui.min.css"/>
+        <link rel="stylesheet" href="https://rsms.me/inter/inter.css"/>
+    </head>
+    <body style="padding-top: 0; margin: 0" class="bg-dark">
+        <header class="bg-dark header-wrap" data-pgc="tts.header">
+            <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+                <a class="navbar-brand header-title hero-text" href="/">That Tech Show</a>
+                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav mr-auto">
+                        <li class="nav-item active">
+                            <a class="nav-link" href="about-us.html">About Us</a>
+                        </li>
+                        <li class="nav-item active">
+                            <a class="nav-link" href="contact-us.html">CONTACT</a>
+                        </li>
+                        <li class="nav-item active">
+                            <a class="nav-link" href="links.html">Links</a>
+                        </li>
+                    </ul>
+                    <a href="https://www.patreon.com/thattechshow" target="_blank" rel="noreferrer nofollow" class="button-link patreon-link"> <svg style="vertical-align: middle" viewBox="0 0 512 512" width="15" height="15">
+                            <path fill="white" d="M512 194.8c0 101.3-82.4 183.8-183.8 183.8-101.7 0-184.4-82.4-184.4-183.8 0-101.6 82.7-184.3 184.4-184.3C429.6 10.5 512 93.2 512 194.8zM0 501.5h90v-491H0v491z"></path>
+                        </svg>
+            Buy us a coffee! </a>
+                </div>
+            </nav>
+            <hr class="hr-rule"/>
+        </header>
+        <main class="main p-3 mb-2 bg-dark text-white" style="padding-left: 15px; padding-right: 15px">
+            <div class="
+          bg-dark
+          d-flex
+          flex-column
+          h-auto
+          justify-content-center
+          pb-5
+          pt-5
+          text-center
+        ">
+                <div class="container pb-4 pt-4" style="max-width: 600px;">
+                    <h2 class="text-uppercase h2">Contact Us</h2>
+                    <div class="mb-3">
+                        {{ site.data.contact.content }}
+                        <a href="mailto:hello@thattech.show" class="text-secondary" target="_pg_blank">hello@thattech.show</a>
+                    </div>
+                    <form action="https://formspree.io/f/moqyllvy" method="POST">
```

Comment:
Let's use Netflify's form. It's much simpler than formspree I think just putting `data-netlify="true"` and a name on the form element.

https://docs.netlify.com/forms/setup/

Reply:
I thought it better to receive an email, than to populate the admin panel, or a spreadsheet
I did set it up to update a google sheet, but decided against that in favour of simplicity of no JS.